src/decision/Criteria/SRMCriteria.jl

```diff
@@ -0,0 +1,69 @@
+"""
```

Comment:
Will this be removed if shading is an environmental layer ? Or is this different?

Reply:
Yes, in fact it's removed in the next set of changes

Reply:
I'm guessing tests will be updated in other changes also?

Reply:
Yes ma'am